cmd/livepeer_cli/livepeer_cli.go

```diff
@@ -74,6 +75,55 @@ type wizard struct {
 	in         *bufio.Reader // Wrapper around stdin to allow reading user input
 }
 
+type wizardOpt struct {
+	desc          string
+	invoke        func()
+	rinkeby       bool
+	transcoder    bool
```

Comment:
Why do we need both `transcoder` and `nonTranscoder`?  Shouldn't we need only 1?

Reply:
we have items that needs to be shown always, items that are shown only in node is transcoder,
and items that shown if node is not transcoder
so, ether we need one flag with three states - but this will lead into introducing new enum, or we need two flags - I've chosen latter one